src/models/gaze17/repairs.md

```diff
@@ -0,0 +1,205 @@
+3. Unplug the white connector that connects the CMOS battery to the motherboard.
+
+![CMOS battery](./img/cmos-battery.webp)
+
+4. To clear the CMOS, [disconnect the main battery](replacing-the-battery), open the lid of the machine, and hold down the power button for at least 15 seconds to discharge any residual energy in the system.
+5. Re-connect the CMOS battery.
+6. [Reconnect the main battery](replacing-the-battery), [replace the M.2 SSD](#replacing-an-m2nvme-ssd), and [replace the bottom cover](#removing-the-bottom-cover).
+7. Power up the Gazelle. The system may power itself off and on after initial boot; this is normal behavior when the CMOS has been reset.
+
+## Replacing the cooling system:
+
+If the CPU/GPU fans become noisy and cleaning them out doesn't fix the issue, you may need a new fan. Contact Support to start a warranty claim or parts purchase.
+
+Thermal paste helps facilitate heat transfer between the CPU/GPU and the cooling equipment. Depending on your climate and the age of the machine, replacing the thermal paste between the CPU/GPU and the heatsink may help the system run cooler.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 20 minutes  
+**Difficulty:** High <span style="color:red;">●</span>  
+
+### Steps to replace the fans/heatsink/thermal paste:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Remove any clear tape that is securing the fan wires.
+3. Remove the silver heatsink screws in the order of the stamped numbers, starting with #1, then #2, and continuing until you have removed #6.
+
+![Thermal screws](./img/thermal-screws.webp)
+
+4. Remove the six silver screws holding the fans onto the chassis in order, highlighted green above.
+    - *Note: Do not remove the smaller black screws holding the fan covers onto the fans.*
+5. Unplug the two white fan connectors from the motherboard (highlighted cyan above).
+6. Remove the heatsink/fans from the case, being careful not to bend the heatsink pipes. It may take some pressure to break the seal of the thermal paste.
+7. Using a paper towel, remove the existing thermal paste. You may also use a small amount of rubbing alcohol if the old paste is dried or difficult to remove.
+
+![Thermal paste removal](./img/thermal-paste-removal.webp)
+
+8. After cleaning the CPU, GPU, and heatsink, apply a small line of thermal paste directly onto the CPU and GPU chips.
+
+![Thermal paste application](./img/thermal-paste-application.webp)
+
+9. Carefully replace the heatsink.
+10. Replace the silver heatsink screws, starting with #1, then #2, and so on until #6.
+11. Replace the silver fan screws.
+12. Plug the two white fan connectors back into the motherboard.
+
+## Replacing the keyboard:
+
+Keyboard replacement is simple and requires only a cross-head screwdriver.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
```

Comment:
Should we mention tweezers as optionally requried in "Tools required" and in the sentence above it? Is a screwdriver even required, or is it just adhesive holding the thing in?

Reply:
Yes, I've copied the verbiage from the oryp8 page over with df22a2805443e1e1abfa9f9df4d59b6aa03ff100.